docs/build.sh

```diff
@@ -160,6 +161,10 @@ rsync -av \
       "${SCRIPT_DIR}"/_ext \
       "${GENERATED_RST_DIR}"
 
+# Merge generated redirects
+# TODO(phlax): Move updating redirects to separate bazel task
+cat /tmp/redirects.txt >> "${GENERATED_RST_DIR}"/redirects.txt
```

Comment:
Does Sphinx just recognize this directly?

Reply:
yep - through the `sphinx.ext.redirafffe` plugin